note/規格書/01-2_行為判定規則.md

```
@@ -126,24 +126,36 @@ LLM 輸出意圖（action / target）
 
 ## 4. 偷竊完整算式（範例示範）
 
-其他行為請照本節格式展開。
+其他行為請照本節格式展開。目前實際落地的只有 `SUCCESS_PARAMS` 的簡化版（僅
+`base + courage × coef`，見 `agent.gd`）。
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)